src/contract.rs

```diff
@@ -310,6 +310,7 @@ pub fn check(
                 }
 
                 state.withdrawable_creation_fee += delta.unwrap();
+                STATE.save(deps.storage, &state)?;
```

Comment:
`state` was not persisted in this function after updating `withdrawable_creation_fee`.